src/ducks/ui/listevisning-selectors.ts

```diff
@@ -107,9 +101,13 @@ export function getMuligeKolonner(filtervalg: FiltervalgModell, oversiktType: Ov
         .concat(
             addHvis(
                 Kolonne.UTLOP_AKTIVITET,
-                !filtervalg.ferdigfilterListe.includes(I_AVTALT_AKTIVITET) &&
-                    harValgtMinstEnAktivitet(filtervalg.aktiviteter) &&
-                    harIkkeValgtTiltakstype(filtervalg.tiltakstyper)
+                //TODO fiks her når aktivitetsfeature fjernes
+                (!filtervalg.ferdigfilterListe.includes(I_AVTALT_AKTIVITET) &&
```

Comment:
this is bit difficult to read..maybe extract some of those conditions in function..

Reply:
yup, fixing it